pkg/cmd/admin/policy/policy.go

```diff
@@ -115,7 +115,7 @@ func getUniqueName(basename string, existingNames *sets.String) string {
 
 // RoleBindingAccessor is used by role modification commands to access and modify roles
 type RoleBindingAccessor interface {
-	GetExistingRoleBindingsForRole(roleNamespace, role string) ([]*authorizationapi.RoleBinding, error)
+	GetExistingRoleBindingsForRole(roleNamespace, role, rb string) ([]*authorizationapi.RoleBinding, error)
```

Comment:
Not sure this parameter makes sense for this method -- also, `s/rb/roleBindingName/`